scripts/components/OpenSearch-Dashboards/build.sh

```diff
@@ -13,11 +13,12 @@ function usage() {
     echo -e "-s SNAPSHOT\t[Optional] Build a snapshot, default is 'false'."
     echo -e "-p PLATFORM\t[Optional] Platform, default is 'uname -s'."
     echo -e "-a ARCHITECTURE\t[Optional] Build architecture, default is 'uname -m'."
+    echo -e "-d DISTRIBUTION\t[Optional] Distribution, default is 'tar'."
```

Comment:
Should the default be `tar.gz` or just `tar` that will be used as extension below?

Reply:
tar, as passed from the run scripts of build.sh.
https://github.com/opensearch-project/opensearch-build/blob/main/src/build_workflow/build_args.py#L19

Reply:
Got it. Thanks!